developer_tests/utilities/parse_args_test.f90

```diff
@@ -0,0 +1,70 @@
+! DART software - Copyright UCAR. This open source software is provided
```

Comment:
where is the test here?

Reply:
in the work dir there is a parse_args_test.in - it needs to run parse_args_test < parse_args_test.in